esmvaltool/diag_scripts/shared/external.R

```diff
@@ -0,0 +1,41 @@
+# Wrappers to call external commands using system2
+# Currently implemented: cdo, nco
+
+cdo <- function(command, args = "", input = "", options = "", output = "",
+                stdout = "", noout = F) {
+  if (args != "") args <- paste0(",", args)
+  if (stdout != "") {
+    stdout <- paste0(" > '", stdout, "'")
+    noout <- T
+  }
+  if (input[1] != "") {
+    for (i in 1:length(input)) {
+      input[i] <- paste0("'", input[i], "'")
+    }
+    input <- paste(input, collapse = " ")
+  }
+  output0 <- output
+  if (output != "") {
+    output <- paste0("'", output, "'")
+  } else if (!noout) {
+    output <- tempfile()
+    output0 <- output
+  }
+  argstr <- paste0(
+    options, " ", command, args, " ", input, " ", output,
+    " ", stdout
+  )
+  print(paste("cdo", argstr))
+  ret <- system2("cdo", args = argstr)
+  if (ret != 0) {
+    stop(paste("Failed (", ret, "): cdo", argstr))
+  }
+  return(output0)
+}
+
+nco <- function(cmd, argstr) {
```

Comment:
It looks like this function is not using nco.

Reply:
The nco wrapper and the cdo wrapper were moved to `esmvaltool/diag_scripts/shared` following @bouweandela 's suggestion in https://github.com/ESMValGroup/ESMValTool/pull/1052#discussion_r282402448
The `extreme_events.R` diagnostic actually uses both of them.

Reply:
What we meant with the comment is that this function has no relation to the nco command whatsoever, it's just a wrapper around system2, so calling it nco is a bit misleading.